sys/aarch64/start.S

```diff
@@ -33,6 +33,8 @@ _ENTRY(_start)
         MOV     sp, x3
 
         BL      aarch64_init
+        /* Temporary stack in VA. */
+        mov     sp, x0
 
```

Comment:
Could you please add a comment somewhere (maybe here) explaining why we need a boot stack in proper virtual addresses for KASAN? I should've left such comment myself and you wouldn't fall into the trap of the kernel exploding while trying to access stack variables.

Reply:
Done.
Please check if it's enough.

Reply:
:+1: